packages/commonwealth/client/scripts/helpers/ContractHelpers/Launchpad.ts

```diff
@@ -16,6 +17,23 @@ class LaunchpadBondingCurve extends ContractBase {
     this.tokenAddress = tokenAddress;
   }
 
+  async launchToken(name: string, symbol: string, walletAddress: string) {
+    if (!this.initialized || !this.walletEnabled) {
+      await this.initialize(true);
+    }
+
+    const txReceipt = await launchToken(
```

Comment:
let's not duplicate "launchToken"

Reply:
do you mean to change the name a bit? - this file actually provides proxy methods to call these contract methods.

Reply:
yeah like handleLaunchToken or something